source/guides/rapids-docker-with-cuda.md

```diff
@@ -0,0 +1,371 @@
+- Additional development utilities
+
+```{note}
+Development components are typically not needed for RAPIDS usage unless you plan to compile CUDA code within your container.
+```
+
+## Getting the Right Components for Your Setup
+
+The [NVIDIA CUDA Container Images repository](https://gitlab.com/nvidia/container-images/cuda) contains a `dist/` directory with pre-built Dockerfiles organized by CUDA version, Linux distribution, and container type (base, runtime, devel).
+
+### Supported Distributions
+
+**Debian/Ubuntu-based (uses `apt`):**
+
+- **Ubuntu**: `ubuntu2004/`, `ubuntu2204/`, `ubuntu2404/`
+
+**RHEL/CentOS-based (uses `yum`/`dnf`):**
+
+- **Rocky Linux**: `rockylinux8/`, `rockylinux9/`, `rockylinux10/`
+- **RHEL/CentOS**: `ubi8/`, `ubi9/`, `ubi10/`
+- **Amazon Linux**: `amzn2023/`
+- **Azure Linux**: `azl3/`
+- **OpenSUSE**: `opensuse15/` (CUDA 13.0+ only)
+
+### Key Differences by Distribution Type
+
+**Ubuntu/Debian distributions:**
+
+- Use `apt-get install` commands
+- Repository setup uses GPG keys and `.list` files
+
+**RHEL/CentOS/Rocky Linux distributions:**
+
+- Use `yum install` or `dnf install` commands
+- Repository setup uses `.repo` configuration files
+- Include repository files: `cuda.repo-x86_64`, `cuda.repo-arm64`
+
+### Installing CUDA components on your container
+
+1. Navigate to `dist/{cuda_version}/{your_os}/base/` or `runtime/` in the repository
+2. Open the `Dockerfile` for your target distribution
+3. Copy all `ENV` variables for package versions and NVIDIA Container Toolkit support
+4. Copy the `RUN` commands for installing the packages.
+5. If you are using the `runtime` components, make sure to copy the `ENV` and `RUN` commands from the `base` Dockerfile as well.
+6. For RHEL-based systems, also copy any `.repo` configuration files needed.
+
+```{note}
+Package versions change between CUDA releases. Always check the specific Dockerfile for your desired CUDA version and distribution to get the correct versions.
+```
+
+### Installing RAPIDS libraries on your container
+
+Refer to the Docker Templates in the [Custom RAPIDS Docker Guide](../custom-docker.md) to configure your RAPIDS installation, adding the conda or pip installation commands after the CUDA components are installed.
+
+## Essential Environment Variables
+
+These environment variables are **required** for CUDA functionality on the container and access to the host system's GPUs through the NVIDIA Container Toolkit
+
+| Variable                     | Purpose                          |
+| ---------------------------- | -------------------------------- |
+| `NVIDIA_VISIBLE_DEVICES`     | Specifies which GPUs are visible |
+| `NVIDIA_DRIVER_CAPABILITIES` | Required driver capabilities     |
+| `NVIDIA_REQUIRE_CUDA`        | Driver version constraints       |
+| `PATH`                       | Include CUDA binaries            |
+| `LD_LIBRARY_PATH`            | Include CUDA libraries           |
+
+## Complete Integration Examples
```

Comment:
IS there a way of templating this example so we don't have to update it every time or that would be tricky?